api/utils.py

```diff
@@ -8,7 +8,7 @@
 from .queries import menu_queries
 from .db_connection import get_db
 
-COLLECTION_PATTERN = r"^(pli-tv-b[ui]-vb|XX|OT|NY|[A-Z]+[0-9]+|[a-z\-]+)"
+COLLECTION_PATTERN = r"^(pli-tv-b[ui]-vb|XX|OT|NG|NK[0-9]+|(?![N])[A-Z]+[0-9]+|[a-z\-]+)"
```

Comment:
This means `NK[0-9]+|(?![N])`
NK with number OR a file that is between brackets. So it seems that the `(?![N])` can go.

On a separate note please also make changes to the regex in the frontend!